### PR TITLE
CDPCP-6883. Update usermanagement.proto

### PR DESCRIPTION
**Pull request by handavid**
Update usermanagement.proto, rebuild and check in build artifacts.
We also need to include options.proto to define the "deprecated"
flag that's used in the usermanagement.proto.

### CODE REVIEW

File: auth-connector/src/main/proto/usermanagement.proto [hunk 1]
```diff
@@ -3893,9 +4057,181 @@ message UpdateUserRequest {
...
+message InteractiveLogoutResponse {
+}
+
+// ========================================================================
+// SECTION: SCIM
+//
+// These messages pertain to SCIM.
+// ========================================================================
+
+message ScimListUsersRequest {
```
Comment (zankhna-randeri) [inline auth-connector/src/main/proto/usermanagement.proto:4102]: Do we also have to copy all defintions for scim?

**Review by zankhna-randeri — COMMENTED** [review]
I don't know whether usermanagement.proto in this repo needs to be a copy of th-usermanagement.proto. So one question about scim calls. 

I assume that everything under generated/main is auto-generated and no need to be review.

**Comment by zankhna-randeri** [comment]
going to add a new state in fmsconverter as a separate pr?

File: auth-connector/src/main/proto/usermanagement.proto (hunk 1 above)
Comment (handavid) [reply auth-connector/src/main/proto/usermanagement.proto:4102]: Yes, it's easier to keep the proto in sync by copying the whole file.

**Review by zankhna-randeri — APPROVED** [review]

**Pull request merged**